docusaurus/docs/reactnative/common-content/core-components/channel/props/deleted_messages_visibility_type.mdx

```diff
@@ -0,0 +1,5 @@
+Controls the visibility of the deleted messages within the channel.
+
```

Comment:
Include the one line description for each of the options here. Similar to this prop in RN docs - https://reactnative.dev/docs/0.65/textinput#importantforautofill-android